scripts/build-android.js

```diff
@@ -87,7 +84,7 @@ for (const arch of architectures) {
     "-GNinja",
     `-DANDROID_NDK=${ndkPath}`,
     `-DANDROID_ABI=${arch}`,
-    `-DCMAKE_MAKE_PROGRAM=${sdkPath}/cmake/${cmakeVersion}/bin/ninja`,
```

Comment:
What's the reason we can rely on the built-in `ninja` and we don't have to use the one provided by the Android SDK?

Reply:
Yes, this way you only have to install ninja once (currently you have to install it twice for local development: node.js + Android)